### ISSUE TITLE
llama-2-7b inversion

### ISSUE THREAD

**Issue by d-bohn** [vec2text/vec2text#50]
Very cool project and I've been having a fun time exploring the repo. I'd like to run some additional examples but I'm having a difficulty reproducing any results using your llama models. It is also a bit difficult to follow how to appropriately load these models if I only wanted to invert text embeddings like you outline with the other models (as opposed to run experiments or evaluate the models).

For example, I have your `t5-base__llama-7b__one-million-instructions__emb` model downloaded along with the `meta-llama/Llama-2-7b-hf` model in my local cache (though it is not clear if this is needed). Using `analyze_utils.load_experiment_and_trainer_from_pretrained` keeps throwing errors on my machine. What would be the easiest way to simply load these model(s) and test embedding and inversion when the models are stored locally (something analogous to the `vec2text.invert_embeddings` and `vect2text.invert_strings` pipeline)?

**Comment by AdrSkapars** [comment]
@d-bohn @victoriazinkovich 

Here is a simple bit of code for inverting one string, that can be done on google colab with 15GB of GPU:

`%%capture
!git clone https://github.com/vec2text/vec2text.git
!pip install vec2text/.

import vec2text

# Load inversion model (includes Llama 2 so dont load that seperately)
inv_model = vec2text.models.InversionFromLogitsEmbModel.from_pretrained("jxm/t5-base__llama-7b__one-million-instructions__emb").to(device)
inv_model.eval()

# Tokenize input you want to get logits fr (I also add a BOS/ pad token and remove the EOS token)
input_ids = [[0] + inv_model.tokenizer("What is the capital of France?").input_ids[:-1]]

# Define input dictionaries
inputs = {"input_ids": input_ids,}
generation_kwargs = {
    "do_sample": False,
    "num_beams": 1,
    "max_new_tokens": len(inputs["input_ids"][0])-1,
}

# This gets logits from encoder and generate using decoder
out = inv_model.generate(inputs=inputs, generation_kwargs=generation_kwargs)

# Print resulting comparison 
print(inv_model.tokenizer.batch_decode(inputs["input_ids"]))
print(inputs["input_ids"])
print(inv_model.tokenizer.batch_decode(out))
print(out.tolist())`

which gets you:

`['<pad> What is the capital of France?']
[[0, 363, 19, 8, 1784, 13, 1410, 58]]
['<pad> What is the capital of France?']
[[0, 363, 19, 8, 1784, 13, 1410, 58]]`

**Comment by jxmorris12** [comment]
Thanks @AdrSkapars - this is super helpful, going to add your snippet to the readme.
